.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Python 3 on Linux
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.6, 3.7]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install --ignore-installed -U -q --no-cache-dir -e .[complete]
```

Comment:
have we checked if `--ignore-installed --no-cache-dir` are needed?

Reply:
`--ignore-installed` definitely isn't given the container used, but I just copied it from our docs to be consistent. `--no-cache-dir` is used as I don't think there is caching in anyway (so why bother even creating a cache dir?). If I'm wrong that would be great.